### PR TITLE
elasticsearch and metastore services for stackfile

### PR DESCRIPTION
**Pull request by zelima**
stackfile updated for auto(re)deploy

### CODE REVIEW

File: docker-cloud.yml [hunk 1]
```diff
@@ -89,4 +89,19 @@ specstore:
   expose:
     - '8000'
   image: 'datopian/specstore:latest'
-
+elasticsearch:
+  autoredeploy: true
+  image: 'elasticsearch:latest'
```
Comment (rufuspollock) [inline docker-cloud.yml]: Is this the elasticsearch which has been deprecated? https://hub.docker.com/_/elasticsearch/

File: docker-cloud.yml [hunk 2]
```diff
@@ -89,4 +89,19 @@ specstore:
...
+  restart: always
+metastore:
+  autoredeploy: true
+  restart: always
+  links:
+    - elasticsearch
+  environment:
+    VIRTUAL_HOST: ${DOMAIN_API}/metastore/*
+    DATAHUB_ELASTICSEARCH_ADDRESS: elasticsearch:9200
+    PRIVATE_KEY:
```
Comment (rufuspollock) [inline docker-cloud.yml]: Why do we have PRIVATE_KEY here? Surely it is PUBLIC_KEY if anything??

File: docker-cloud.yml (hunk 1 above)
Comment (zelima) [inline docker-cloud.yml]: No this official elasticsearch image

File: docker-cloud.yml (hunk 2 above)
Comment (zelima) [inline docker-cloud.yml]: Not sure why, but os-condactor search uses PRIVATE_KEY
https://github.com/openspending/os-conductor/blob/master/conductor/blueprints/search/blueprint.py#L5

Comment (rufuspollock) [inline docker-cloud.yml]: @akariv can we get your input here - why is PRIVATE_KEY being used in search.

Comment (rufuspollock) [inline docker-cloud.yml]: Apparently private key **is** needed.

**Comment by rufuspollock** [comment]
Will replace with a new commit.

**Pull request closed without merging**